buổi-9/bai2/bai-2.js

```diff
@@ -0,0 +1,10 @@
+function test() {
+    var age=window.prompt('Nhập tuổi học sinh: ')
+    if(age<16){
```

Comment:
nhìn kỹ cách viết dấu cách
```suggestion
    if (age < 16) {
```